scripts/bb_sim.py

```diff
@@ -50,7 +50,7 @@
     )
 
     try:
-        args = parser.parse_known_args()[0]
+        args = parser.parse_args()
```

Comment:
Why?

Reply:
reverting back to raising error when unexpected parameters provided.
so when the user accidentally provided typo or anything incorrect, error will be raised;
instead of just running it through it default values and never realize its incorrect.